### DISCUSSION TITLE
Powering The Rover

### DISCUSSION THREAD

**Discussion by conner-replogle** [Comet-Robotics/URC#16] [PCB/Power]
We will have many devices to power including
- [OptiPlex 7050](https://www.amazon.com/Dell-OptiPlex-7050-Micro-Bit-Multi-Language/dp/B08GCX4JKJ/)
- [4 CIM Motors](https://www.vexrobotics.com/217-2000.html)
- D435 Realsense

All of this needs to be powered on the rover. We need to figure out voltage size, and continuous amp delivery for the battery we need for powering all of it for prolonged period of time.

Then we need to figure out the actual implementation of powering all of them with the right connectors etc.

**Comment by conner-replogle** [comment]
according to my calculations the 
Runtime: 1hr

4 CIM Motors: 1605Wh at half power

OptiPlex: 65Wh

I'd say target of around 2000wh
[Battery 24v 80ah](https://www.amazon.com/OCELL-80Ah-LiFePO4-Off-Grid-Trolling/dp/B0BLH46WH8?source=ps-sl-shoppingads-lpcontext&smid=APQJKPSY673C3&th=1)

**Comment by PumpkinClouds** [comment]
I found this deal for 4 12V 100 Ah batteries which could help reduce step down inefficiencies: [https://www.amazon.com/LiFePO4-Lithium-Battery-Batteries-Trolling/dp/B0CLZQJK8Y/ref=sr_1_4?dib=eyJ2IjoiMSJ9.K7H_5UhVCsIRj4kd7CTWu9BTj-CDNqHT9t6Vtbxz2KT59b3BHKcJwEq7TWAu14UTAv4WMtqggSqQtCa6cIvtuOkUXsAVOXbwgQUGOqSK6CcRBoOS7rmuOzpyPAwY-O3mFlTa2m-_RogNhDAI1Vcc8wpq2iE_tTkcr4VA5q9saxSdbwuSnQSnYdMDhQO8eJXDEpAhEcVljarZl12sdO4wJDcz6OsqA6Qmnto2XW8Vc2w.DK_Wh4PT9JU-10vU6VJlLFN_5zwaPwH1jyqbvc7EXcI&dib_tag=se&keywords=48v%2Blithium%2Bion%2Bbattery&qid=1726101779&sr=8-4&th=1]

**Comment by pablo-collantes** [comment]
Gonna revive this post a bit since we really need to just purchase a battery.

Issues with wiring 12V batteries of high amp hour in Series: they typically have built in Battery Management Systems, however these BMS' do not regulate power levels across all batteries so the second the 1st battery dies the rest of the batteries cannot be used.

Issues with wiring 24V batteries on Parallel:
[reddit discussion](https://www.reddit.com/r/batteries/comments/ebjly0/safe_way_to_connect_two_bms_protected_liions_in/)
In that discussion they go over the issues of separate batteries with BMS' attempting to charge each other when one dies resulting in high current flows between one another and opening BMS' when they attempt to shut off due to low voltage protection.
They go over some potential solutions the ones I thought were the best were 
- creating a main battery and reserve battery switch when main dies quickly move to reserve
- Placing a high amp diode between batteries to prevent back charge however this would result in a lot of power lost to the diode and would most likely require a heatsink.

Final Thoughts:
- connecting 24V batteries in parallel and create a main/reserve battery switch 
- Bite the bullet and purchase expensive 24V 80-100 amp hour battery to avoid extra work

**Comment by JonnyJonny2000** [comment]
[DC 24v 100Ah $369 44.8lb ](https://www.amazon.com/DC-HOUSE-24V-Battery-Trolling/dp/B0D91ZSPYX?source=ps-sl-shoppingads-lpcontext&ref_=fplfs&smid=A1P855JR5KPOWZ&th=1)

**Comment by PondDuck777** [comment]
Battery Stats
Cell: Molicel 21700 P42A 4200mAh 45A Battery
Nominal Volt: 3.6 V
Capacity: 4200 mAh
Discharge rate: 30 amps (motors and other components running at the same time)
Drawing watts in terms of 30 amps: 720 W
Series: 7 Cells        Parallel: 6 Cells
Total Cells: 42
Capacity (Ah): 25.2 Ah
Total Cost: $165.9
Weight (lb): 6.48
